Cargo.toml

```diff
@@ -44,3 +44,7 @@ serde = { version = "1.0", features = ["derive"] }
 strum = { version = "0.24", features = ["derive"] }
 envy = "0.4.2"
 which = "4.4"
+
+[patch.crates-io.embuild]
+git = "https://github.com/haileys/embuild"
+branch = "esp-idf-no-git-repo"
```

Comment:
just for integration purposes for now, I'll rebase this change away before merge